### DISCUSSION TITLE
Should HTML Caption work?

### DISCUSSION THREAD

**Discussion by djdowning** [quarto-dev/quarto-cli#5695] [Q&A] [answered]
```
---
title: "Test Plotly"
format: html
keep-md: true
---

Testing with Knitr_1.43.1

```{r}
library(plotly)
fig <- plot_ly(midwest, x = ~percollege, color = ~state, type = "box")
fig
```

Quitting from lines 10-13 [unnamed-chunk-1] (test.qmd)
Error in `add_html_caption()`:
! unused argument (xfun::grep_sub("^[^<]*<[^>]+aria-labelledby[ ]*=[ ]*\"([^\"]+)\".*$", "\\1", x))

Using with Knitr_1.43.1. 

Thank you so much for your help.

**Comment by mcanouil** [comment]
Your example is not complete/valid and currently the title of the thread has nothing to do with your post.

Could you please rewrite your post (title and description) to make it clear what it is about and what is your issue?

Could you share a small "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````

**Comment by mcanouil** [comment, marked as answer]
Very likely related to #5702.
I am closing this, please follow on the issue.
Thanks for reporting this.
